deploy/backup/copy-backup.sh

```diff
@@ -35,10 +35,25 @@ get_backup_dest() {
     local backup=$1
 
     if $ctrl get "pxc-backup/$backup" 1>/dev/null 2>/dev/null; then
+        BASE64_DECODE_CMD=""
+        echo $(echo 'abc'|base64) | base64 -d 1>/dev/null 2>/dev/null
+        res1=$?
+        echo $(echo 'abc'|base64) | base64 -D 1>/dev/null 2>/dev/null
+        res2=$?
+
+        if [[ ${res1} == 0 ]]; then
```

Comment:
the following way looks a bit simplier
```
if echo eWVz | base64 -d 1>/dev/null 2>/dev/null; then
    BASE64_DECODE_CMD="base64 -d"
elif echo eWVz | base64 -D 1>/dev/null 2>/dev/null; then
    BASE64_DECODE_CMD="base64 -D"
else
    echo "base64 decode error."
    exit 1
fi
```

Reply:
All right, I have modified the code again.